source/documentation/partials/_messagetypeintro.md

```diff
@@ -1,4 +1,20 @@
-## Accessing the XML Schemas
+## Message details
+
+This section provides the following information about each message type: 
+
+- Characteristics of the data groups belonging to the message: sequence, number of repetitions, and a status value to indicate if the data group is: 
```

Comment:
Two issues:
- Is "belonging to a message" better than "belonging to the message"?
- Is "if a data group" better than "if the data group"?

The concern here is the use of the definite article instead of the indefinite article. The first part of the sentence says "the data groups", which suggests many data groups not a singe data group. The wording of the sentence does not hang together.

Suggestion: Characteristics of the data groups belonging to a message: sequence, number of repetitions, and a status value to indicate if a a data group is: